src/components/CheckBoxList/check-box-list-type.ts

```diff
@@ -15,6 +15,8 @@ export interface CheckBoxListItemSxProps {
     checkboxList?: SxProps;
     checkboxButton?: SxProps;
     checkbox?: SxProps;
+    dragAndDropContainer?: SxProps;
+    list?: SxProps;
```

Comment:
Rename it checkboxList and rename existing `checkboxList` to `checkboxListItem` which is used by items `CheckboxListItem` or `DraggableCheckBoxListItem`

Reply:
done